test/runtests.jl

```diff
@@ -61,6 +61,11 @@ include(joinpath(examples_path, "kinsol_mkinTest.jl"))
 @test abs(minimum(residual)) < 1e-5
 end
 
+println("== handle tests")
+let
+@testset "Handle Tests" begin include("handle_tests.jl") end
```

Comment:
I think it's a separate PR, because we should really want to cleanup those let blocks at the same time, actually test values on the other examples, stop printing so many times in the common tests, etc. There's a lot to do here, so for now I semi-matched the old style.